interface/management/commands/enrich.py

```diff
@@ -151,15 +147,17 @@ def handle(self, *args, **options):
             logger.info("Got {} new tasks for enrichment.".format(len(tasks)))
 
             for task in tasks:
-                data = self.get_data(task)
-                self.mark_task_as_running(task)
-                ptasks = task.plugintask_set.filter(status=STATUS_NEW)
-                task_queue = self.make_ptask_queue(ptasks)
-                logger.info("Will run queue of {} plugins for enrichment.".format(len(ptasks)))
-                final_data = self.run_ptask_queue(data, task_queue, ptasks)
-                logger.info("Writing results to database.")
-
                 try:
+                    data = self.get_data(task)
+                    self.mark_task_as_running(task)
+                    PluginTask(task=task, plugin_name='GeoPlugin').save()
```

Comment:
Why are you explicitly calling the GeoPlugin name here? If I remember it correctly, the plugin should already be called by `make_ptask_queue` and `run_ptask_queue`.

I fear that explicitly calling the plugin may break things if the plugin is disabled in the conf.